### PR TITLE
Remake Convert2RHEL

### PR DESCRIPTION
**Pull request by Lennonka**
Remake of the whole assembly for Sat 6.12

I suggest to reconsider if the upstream still wants it as a separate guide since it's much shorter now. For Satellite, the assembly was moved to _Managing Hosts_, which seems to be a more appropriate place in the docs structure. (Previously in _Managing Content_.) I would personally cancel the separate guide.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

**Comment by Lennonka** [comment]
We're waiting for the completion of https://github.com/theforeman/foreman-ansible-modules/pull/1467

**Comment by maximiliankolb** [comment]
I like the standalone guide; the current status seems to break upstream. If you have good arguments to move it to "managing hosts", then I'd be okay with that too.

**Comment by Lennonka** [comment]
Okay, I'll try to make my case:

- PRO The increasing number of standalone guides makes the orientation in the docs increasingly difficult.
- CON However, the number of chapters in _Managing Hosts_ does not improve the situation much. Perhaps this guide should be split into two somehow.
- PRO The chapter is now much simpler and shorter, is it really worthy of a standalone guide?

Let me know if these arguments are sufficient. If not, let's keep it standalone.

Well, the thing is that the overall information architecture of the docs would deserve an overhaul, but I don't think that anybody has the capacity to do that.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+The Convert2RHEL utility in {Project} consists of an Ansible role and an Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, repositories, products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can convert:
+
+* CentOS 7 to {RHEL} 7
+* Oracle 7 to {RHEL} 7
+* CentOS 8 to {RHEL} 8
+* Oracle 8 to {RHEL} 8
```
Comment (ekohl) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:16]: I think they're named CentOS Linux and Oracle Linux.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Must be set to `false`
+| `{project-context}_validate_certs` * | boolean | Set to `true` if you wish to enable certificate checks in Ansible. (default: `true`)
```
Comment (ekohl) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:20]: Why is this marked with an asterisk? From the description above it sounds like it's required but IMHO this shouldn't be required.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| `{project-context}_convert2rhel_manage_subscription` | boolean | Set to `false` if you already have a manifest on your {ProjectServer}. If you decide to upload a new manifest from disk, the current manifest will be overwritten. (default: `true`)
+| `{project-context}_content_rhel_enable_rhel7` | boolean | Enables {RHEL} 7 repositories. Set to `true` if you intend to convert hosts to {RHEL} 7.
+| `{project-context}_convert2rhel_enable_centos7` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 7. Otherwise must be set to `false`.
```
Comment (ekohl) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:23]: Why is this required? Feels like you should be able to get this from the host data in Foreman itself.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+These variables are marked with an asterisk * in the table below.
+You must create those additional variables manually and assign them to the `{ansible-namespace}.convert2rhel` role.
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
```
Comment (ekohl) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:17]: Is username/password the only authentication? That feels like a bad UX and possibly very insecure. Foreman has support for API tokens so are they supported?

**Review by ekohl — COMMENTED** [review]
In general I do think Managing hosts is way too long, but I also see that this is now quite short. Thinking about it, I wonder if we should have the chapter 1, but make the table of variables an appendix. That does depend on how many variables must really be set.

**Comment by maximiliankolb** [comment]
I got feedback from Foreman/orcharhino users saying that they very much prefer shorter guides.

"breaks upstream" > yes, because content seems to have moved/is not included correctly. Only after this PR, the standalone variant would be (too) short.

regarding navigation: I was hoping someone with UX experience is eventually willing for rework the navigation. I think some guides have rather bad names, for example "Configuring Foreman to use Ansible" could be renamed to "Managing Hosts using Ansible"; and the Salt part of "Managing Hosts" could be moved to a standalone guide.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:20]: What I tried to say with the asterisk was this: If there isn't an asterisk, the user can just edit the variable because it was imported with the role. If there is an asterisk, the user has to create the variable first to configure it, because it wasn't imported with the role. I.e. the asterisk was supposed to say that the user has to use slightly different steps. But if it's confusing then I should probably just leave it out.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc (hunk 3 above)
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:23]: See above.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:20]: Required/optional is given by the table in which the variable is mentioned.

Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:20]: Is it really that confusing? :D

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc (hunk 4 above)
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:17]: Yes, only login/pass is supported. I checked with Leos.

**Comment by Lennonka** [comment]
@maximiliankolb So can I move it to the admin guide for non-Satellite? Perhaps we could raise an issue for the restructuring and discuss there what could be improved. Can you kick it off, please?

**Comment by Lennonka** [comment]
@ekohl We actually try to avoid appendixes and keep topics together.

**Comment by ekohl** [comment]
Big :+1: from me about rethinking the navigation. When I restructured the current navigation I was very clear in that I thought it was only a first step.

**Comment by maximiliankolb** [comment]
@Lennonka Yes, go ahead and move it. Please ping me once your PR is ready for review.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:20]: Okay, reword might be a good idea.

I was told that they are not imported due to the way Satellite imports Ansible variables. I have no fraking idea what that means. However, I did create a BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2126168
Not sure if I assigned it to the correct people. Can you pls check and fix it if needed?

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Must be set to `false`
```
Comment (stejskalleos) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:19]: I wouldn't say `must be set`, it can be set to `true` but it kinda doesn't make sense for user to do it

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+| `{project-context}_convert2rhel_enable_centos8` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 8. Otherwise must be set to `false`.
+| `{project-context}_convert2rhel_enable_oracle8` | boolean | Set to `true` if you want to prepare conversion data for Oracle 8. Otherwise must be set to `false`.
+|====
+
+.Optional variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_manifest_path` * | string | Path to a manifest to upload from disk, such as `~/manifest.zip`. The path must be set if you upload a new manifest from disk with `{project-context}_convert2rhel_manage_subscription` set to `true`.
+| `{project-context}_content_rhel_rhel8_releasever` * | string | Minor release version, such as `8.4`. (default: latest)
```
Comment (stejskalleos) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:35]: When I was trying to the conversion, I had problem with RHEL version,
doing conversion with `latest` repos didn't work, I had to enable `8.5` repos, then it worked.
See https://access.redhat.com/discussions/5676681

So maybe we should encourage users to set `_content_rhel_rhel8_releasever` by default to `8.5`, or atleast mention the issue here.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+endif::[]
+
+The amount of time the conversion process takes can vary depending on how many packages need to be replaced, network speed, storage speed, and similar factors.
+
+.Prerequisites
+* Review https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel[Supported conversion paths] in _Converting from an RPM-based Linux distribution to RHEL_.
+ifdef::satellite[]
+* You must have completed the steps 1.{range}5. of the procedure https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel[Preparing for a RHEL conversion] in _Converting from an RPM-based Linux distribution to RHEL_.
+endif::[]
+* Ensure you have a subscription manifest uploaded to your {Project} and that there are sufficient {RHEL} entitlements allocated for the conversions you intend.
```
Comment (stejskalleos) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:36]: I would mention here that it can be also done also by the `convert2rhel` role

**Review by stejskalleos — CHANGES_REQUESTED** [review]

**Comment by maximiliankolb** [comment]
@stejskalleos You can either download the built docs from github UI (check tab "checks") or read the built docs on surge.sh (see top comment).

**Comment by stejskalleos** [comment]
Good to know that we have online version.
Checked it and it looks fine, so let's get this in.

Thanks @Lennonka @maximiliankolb @ekohl !

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 8]
```diff
@@ -0,0 +1,75 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the Convert2RHEL utility to simplify the conversion process.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:5]: Maybe some form of highlighting for "Convert2RHEL"?

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can convert:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
+
+You cannot convert a version 7 system into {RHEL} 8, or a version 8 system into {RHEL} 7.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:18]: Is it really necessary to explain what it _cannot_ do?

Can you convert Rocky Linux 8 to RHEL8? AlmaLinux?

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
+Third-party packages and non-{Team} packages that are not available in {RHEL} are retained.
+
+The Convert2RHEL utility removes unnecessary packages such as logos, or packages known to cause issues during the conversion.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:23]:
```suggestion
The Convert2RHEL utility removes unnecessary packages such as logos or packages known to cause issues during the conversion.
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+The utility replaces the `CentOS-release` or `Oracle-release` package with the `rhel-release` package, and all packages signed by CentOS or Oracle with their {Team} equivalents.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:24]:
```suggestion
The utility replaces the `CentOS-release` or `Oracle-release` package with the `rhel-release` package, and all packages signed by CentOS or Oracle with their Red Hat equivalents.
```

There are no Foreman/Katello/ATIX distributions.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+ifdef::satellite[]
+The utility also subscribes the host to {Team} Subscription Management.
+endif::[]
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:27]:
```suggestion
The utility also subscribes the host to {Team} Subscription Management.
```

I assume this is necessary for all builds; otherwise, RHEL won't receive any content.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+The amount of time the conversion process takes can vary depending on how many packages need to be replaced, network speed, storage speed, and similar factors.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:29]:
```suggestion
The duration of the conversion process depends on the number of packages that have to be replaced, network speed, storage speed, and similar factors.
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+
+.Prerequisites
+* Review https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel[Supported conversion paths] in _Converting from an RPM-based Linux distribution to RHEL_.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:32]: Maybe for non-satellite build targets, you add "Ensure the OS of your managed host is supported by the convert2rhel utility." and make this satellite only.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+ifndef::orcharhino[]
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:33]:
```suggestion
ifdef::satellite[]
```

What's the alternative for non-Satellite?

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 16]
```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:2]: So am I! I'm trying to ask Shweta for her opinion. It may take some time.

[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:2]: This can be changed later on if required ;)

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Wait until repository sync finishes before continuing with data generation. Set to `false`.
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:19]: This is not a default, this tells the user to set the value to false. I'll rephrase it.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 18]
```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
+
+Before you run the Ansible role to generate conversion data, configure values of the following required Ansible variables.
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:4]: Generates data. The conversion is done in another step.

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+
+{Project} imports most of the required Ansible variables from the `{ansible-namespace}.convert2rhel` role.
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:6]: Some are not imported probably by mistake, and have to be created manually.
See https://bugzilla.redhat.com/show_bug.cgi?id=2126168
Not sure how to handle this.

[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:6]: The ones that ARE imported do not have the asterisk next to them in the tables.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc (hunk 15 above)
Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:33]: Why can't we mention the official docs for the tool for non-Satellite?

Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:33]: Oh, hang on. There are man pages in the tool that might be an alternative.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc (hunk 9 above)
Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:18]: The situation is this: The listed conversion paths are officially supported by RH. There are other possible conversion paths, which are described in an article that is referenced in Additional resources below.
I wonder if we should mention those links for non-Satellite as well, because users eventually need to access RH resources anyway, such as for the subscription manifest.

Comment (maximiliankolb) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:18]: Maybe reword to "If you want to convert EL7 to EL8, see LEAPP" which I think is documented for Foreman Server? Otherwise, leave as is; I might create a follow-up PR to extend it slightly.

Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:18]: The CLI utility can convert other systems, but the Ansible role and playbook provides only the listed conversions. (Verified with developers.)

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Wait until repository sync finishes before continuing with data generation. Set to `false`.
+| `{project-context}_validate_certs` * | boolean | Set to `true` if you wish to enable certificate checks in Ansible. (default: `true`)
+| `{project-context}_convert2rhel_manage_subscription` | boolean | Set to `false` if you already have a manifest on your {ProjectServer}. If you decide to upload a new manifest from disk, the current manifest will be overwritten. (default: `true`)
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:21]: I wouldn't put it here. The link is mentioned in the top-level steps in the concept module.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc (hunk 9 above)
Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:18]: Seriously, I wouldn't complicate this and go just with the listed conversion paths as supported by the tool.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc (hunk 15 above)
Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:33]: Why can't we link to the official docs of the utility for all products? If the user uses RHEL for hosts, they need to access the official docs eventually regardless of the "Foreman" product.

Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:33]: @maximiliankolb I don't understand your reasoning, can you please enlighten me on that?

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 21]
```diff
@@ -0,0 +1,85 @@
...
+The *Convert2RHEL* utility in {Project} consists of an Ansible role and Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, enables required repositories and creates products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can use the Ansible role to generate conversion data for the following conversions:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
```
Comment (Lennonka) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:16]: @maximiliankolb When I mention other systems, then the problem is that the General Steps no longer apply and an alternative approach must be taken.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+* You must have completed the steps 1.{range}5. of the procedure https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel[Preparing for a RHEL conversion] in _Converting from an RPM-based Linux distribution to RHEL_.
+* Ensure you have a subscription manifest uploaded to your {Project} and that there are sufficient {RHEL} entitlements allocated for the conversions you intend.
+Alternatively, you can use Ansible variables to tell the role to import the manifest from disk.
+The manifest must be imported to the organization to which you will register hosts for conversion.
++
+You can update your allocations and download the updated manifest from the https://access.redhat.com[Red Hat Customer Portal].
+ifndef::orcharhino[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using manifests] in _Red Hat Subscription Management_.
+endif::[]
```
Comment (Lennonka) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:43]: @maximiliankolb I'm not sure if this should be included for orcharhino as well or not. Pls let me know.

**Comment by Lennonka** [comment]
@maximiliankolb @stejskalleos How is it working now?

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 23]
```diff
@@ -0,0 +1,74 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the *Convert2RHEL* utility to simplify the conversion process.
+
+The *Convert2RHEL* utility in {Project} consists of an Ansible role and Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, enables required repositories and creates products, activation keys, and host groups.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:8]:
```suggestion
You use the Ansible role to generate conversion data on {ProjectServer}, which includes required repositories and creates products, activation keys, and host groups.
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 24]
```diff
@@ -0,0 +1,74 @@
...
+. Import the `{ansible-namespace}.convert2rhel` Ansible role and variables.
+For more information, see {ConfiguringAnsibleDocURL}importing-Ansible-roles_ansible[Importing Ansible Roles and Variables] in _{ConfiguringAnsibleDocTitle}_.
+. Configure Ansible variables for generation of conversion data.
+For more information, see xref:variables-for-generation-of-conversion-data_{context}[].
+. Assign the `{ansible-namespace}.convert2rhel` role to the host that represents {ProjectServer}.
+For more information, see {ConfiguringAnsibleDocURL}adding-ansible-roles-to-an-existing-host_ansible[Assigning Ansible Roles to an Existing Host] in _{ConfiguringAnsibleDocTitle}_.
+. Run the Ansible role on {ProjectServer}.
+For more information, see {ConfiguringAnsibleDocURL}running-ansible-roles-on-a-host_ansible[Running Ansible Roles on a Host] in _{ConfiguringAnsibleDocTitle}_.
++
+The Ansible role generates data required for host conversion, that is, repositories, certificates, activation keys and host groups.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:54]:
```suggestion
The Ansible role generates data required for host conversion, that is, repositories, certificates, activation keys, and host groups.
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+The role enables the `{RepoRHEL7Server}` repository with the 7Server release and x86_64 architecture, or `{RepoRHEL8BaseOS}` and `{RepoRHEL8AppStream}`, or both, depending on which variables you have set in the previous steps.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:55]: I feel like "7Server" deserves some kind of highlighting, as does "x86_64".

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Set to `false` if you do not want {ProjectServer} to wait until repository sync finishes before continuing with data generation.
+| `{project-context}_validate_certs` * | boolean | Set to `true` if you want to enable certificate checks in Ansible. (default: `true`)
+| `{project-context}_convert2rhel_manage_subscription` | boolean | Set to `false` if you already have a manifest on your {ProjectServer}.
+If you upload a new manifest from disk, the current manifest will be overwritten. (default: `true`)
+| `{project-context}_content_rhel_enable_rhel7` | boolean | Enables {RHEL} 7 repositories.
+Set to `true` if you intend to convert hosts to {RHEL} 7.
+| `{project-context}_convert2rhel_enable_centos7` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 7.
+Otherwise, you must set the value to `false`.
+| `{project-context}_convert2rhel_enable_oracle7` | boolean | Set to `true` if you want to prepare conversion data for Oracle 7.
```
Comment (maximiliankolb) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:27]:
```suggestion
| `{project-context}_convert2rhel_enable_oracle7` | boolean | Set to `true` if you want to prepare conversion data for Oracle Linux 7.
```

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+Otherwise, you must set the value to `false`.
+| `{project-context}_content_rhel_enable_rhel8` | boolean | Enables {RHEL} 8 repositories. Set to `true` if you intend to convert hosts to {RHEL} 8.
+| `{project-context}_convert2rhel_enable_centos8` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 8.
+Otherwise, you must set the value to `false`.
+| `{project-context}_convert2rhel_enable_oracle8` | boolean | Set to `true` if you want to prepare conversion data for Oracle 8.
```
Comment (maximiliankolb) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:32]:
```suggestion
| `{project-context}_convert2rhel_enable_oracle8` | boolean | Set to `true` if you want to prepare conversion data for Oracle Linux 8.
```

File: guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc [hunk 28]
```diff
@@ -0,0 +1,44 @@
...
+Otherwise, you must set the value to `false`.
+|====
+
+.Optional variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_manifest_path` * | string | Path to a manifest to upload from disk, such as `~/manifest.zip`.
+The path must be set if you upload a new manifest from disk with `{project-context}_convert2rhel_manage_subscription` set to `true`.
```
Comment (maximiliankolb) [inline guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc:41]:
```suggestion
Set this path to `true` if you upload a new manifest from disk with `{project-context}_convert2rhel_manage_subscription`.
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 29]
```diff
@@ -0,0 +1,74 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the *Convert2RHEL* utility to simplify the conversion process.
+
+The *Convert2RHEL* utility in {Project} consists of an Ansible role and Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, which includes enabling required repositories and creating products, activation keys, and host groups.
```
Comment (Lennonka) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:8]: @maximiliankolb I tried to word it differently than you were suggesting, pls check.

Comment (maximiliankolb) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:8]: Sounds good to me!

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by Lennonka** [comment]
Cherry-picked:
- b9100a4a3..994a4cc56  3.4 -> 3.4
- ae4dab747..a5cc49e44  3.3 -> 3.3
